### DISCUSSION TITLE
Microsoft Store Version Installation Error

### DISCUSSION THREAD

**Discussion by Gvolten** [emoacht/Monitorian#650] [General]
Hi, I was using the app from Microsoft Store until recently, when it stopped working for whatever reason. I tried uninstalling it, but it is stuck in the Windows menu like this:
![image](https://github.com/user-attachments/assets/733adb94-3258-4e88-8921-f68d8d7721c5)

When I go to Microsoft Store, it shows it as not installed. It also does not appear in the system settings programs list. If I try to install it, I get the error code `0x80073CF9` and it just fails.

I tried installing the version with the installer, which worked, but it did not remove the Microsoft Store version. I now have two apps. One from the store (not working) and one from the installer (functional). I checked and the files for the Microsoft Store version are still there (In `C:\Program Files\WindowsApps` - I checked in safe mode) . Can I get rid of it completely somehow?
![image](https://github.com/user-attachments/assets/8de90b11-d901-4e57-b295-0874ec32e32d)

**Comment by emoacht** [comment]
Thank you for reporting.
But the installation process from Microsoft Store app is out of control of a developer.
The installation by WinGet seems to be more reliable.

**Comment by Gvolten** [comment]
Good, how do I remove the Microsoft Store version from my PC?

**Comment by emoacht** [comment]
Nothing special. Uninstall it from right click menu.

**Comment by Gvolten** [comment]
I can't. It stays there no matter how I try to delete it. In the Store it says it is not installed and when I try to install it, I get the error mentioned earlier.

**Comment by emoacht** [comment]
You can try to uinstall by Winget.
```
winget uninstall monitorian
```

**Reply by Gvolten** [reply]
This does not work unfortunately.
First, it uninstalled both versions. Now it says it is uninstalling the MS Store version, but it is still present and not functional. Did the uninstaller get lost or something?
![image](https://github.com/user-attachments/assets/9d1d1533-3df0-4ae3-a965-8c9f6961c459)

![image](https://github.com/user-attachments/assets/bcc3f093-95ca-4377-8ea2-c72746948c81)

**Comment by Gvolten** [comment]
It looks like I have been subjected to this issue: https://www.reddit.com/r/WindowsHelp/comments/1gupjaq/the_solution_on_how_to_fix_microsoft_store_apps/

I used the first, then the second, and then the first solution again to fix the problem.
Sorry for wasting your time, @emoacht , and thanks for trying to help even though it clearly wasn't an issue on your side.
